### ISSUE TITLE
Implement issue importance

### ISSUE THREAD

**Issue by harmonyrose** [harmonyrose/3rd-Party-Problem#16]
Some issues are more important to voters than others. Adding issue importance allows us to add a "single-issue voter" algorithm, and perhaps modify some CI2 mechanisms i.e. agents are more likely to talk about issues that are important to them.

**Comment by divilian** [comment]
Agreed, but this is a placeholder for the (post-ANNSIM) future, right?

**Comment by divilian** [comment]
Maybe, but also see Kottonau and Pahl-Wostl 2004.
